web/bfex/common/schema.py

```diff
@@ -23,3 +23,19 @@ class FacultySchema(Schema):
     def _create_faculty(self, data):
         """Turns the extracted json data into an instance of Faculty"""
         return Faculty(meta={'id': data["faculty_id"]}, **data)
+
+class KeywordSchema(Schema):
+    """Marshmallow schema used for validating Faculty JSON objects.
+
+    A marshmallow schema allows us to easily extract information from JSON input, while at the same time, performing
+    basic validation of that data.
+    """
+    faculty_id = fields.Integer(load_from="id", required=True)
+    rake_keywords = fields.String()
+    generic_keywords = fields.String()
+
+    @post_load
+    def _create_faculty(self, data):
+        """Turns the extracted json data into an instance of Faculty"""
+        return Keywords(meta={'id': data["faculty_id"]}, **data)
```

Comment:
This line will break. A keyword model is not identified by the faculty_id, it is identified by the faculty_id, approach_id, and datasource. Look at #37 to see how it is stored in elastic.